src/main/java/org/commcare/formplayer/services/RestoreFactory.java

```diff
@@ -173,49 +176,56 @@ public void configure(AuthenticatedRequestBean authenticatedRequestBean, HqAuth
         this.configured = true;
         sqLiteDB = new UserDB(domain, scrubbed_username, asUsername);
         log.info(String.format("configuring RestoreFactory from authed request with arguments " +
-                "username = %s, asUsername = %s, domain = %s, useLiveQuery = %s",
+                        "username = %s, asUsername = %s, domain = %s, useLiveQuery = %s",
                 username, asUsername, domain, useLiveQuery));
     }
 
     // This function will only wipe user DBs when they have expired, otherwise will incremental sync
     public UserSqlSandbox performTimedSync() throws Exception {
-        return performTimedSync(true, false);
+        return performTimedSync(true, false, false);
     }
-    public UserSqlSandbox performTimedSync(boolean shouldPurge, boolean skipFixtures) throws Exception {
+
+    public UserSqlSandbox performTimedSync(boolean shouldPurge, boolean skipFixtures, boolean responseTo412) throws Exception {
```

Comment:
'response' is a verb, so I found this name slightly confusing, can you change it to "isResponseTo412"?